packages/core/src/stylable-meta.ts

```diff
@@ -61,9 +62,13 @@ export class StylableMeta {
     // Generated during transform
     public targetAst?: postcss.Root;
     public globals: Record<string, boolean> = {};
-    constructor(public sourceAst: postcss.Root, public diagnostics: Diagnostics) {
+    constructor(
+        public sourceAst: postcss.Root,
+        public diagnostics: Diagnostics,
+        flags: FeatureFlags
```

Comment:
make it a member. this way we can debug it when we see the meta. also I think per meta flags